specification/dimensions/chargedescription.md

```diff
@@ -0,0 +1,30 @@
+# Charge Description
+
+A Charge Description provides a high-level context of a line item without requiring additional discovery.  This column is typically a brief, human-readable summary of a select group of corresponding details across a cost and usage dataset or provides information not otherwise available.
+
+The Charge Description column MUST be present within billing data, MUST be of type String, and SHOULD NOT be null.  Providers SHOULD specify the length of this column in their publicly available FOCUS documentation.
```

Comment:
The length thing is a bit interesting, perhaps it would be good as a spec to just set an upper bound. I've never seen anything above 200 chars in the field.

Reply:
SaaS descriptions are MUCH longer given that the description contains details that are not otherwise part of the an invoice, so I'd be hesitant to define a static length, and instead let providers do so.